content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,595 @@
+| `-a` | `--account=<organization>` | Specify an Upbound organization during authentication. |
+{{< /table >}}
+
+
+#### Authentication options
+
+The `crossplane xkpg login` command can use a username and password or Upbound API token.
+
+Provide a username and password with the `--username` and `--password` flags or
+set the environmental variable `UP_USER` for a username or `UP_PASSWORD` for the
+password. 
+
+Use an Upbound user token instead of a username and password with `--token` or 
+the `UP_TOKEN` environmental variable. 
+
+{{< hint "important" >}}
+The `--token` or `UP_TOKEN` environmental variables take precedence over a 
+username and password.
+{{< /hint >}}
+
+Using `-` as the input for `--password` or `--token` reads the input from stdin.  
+For example, `crossplane xpkg login --password -`.
+
+After logging in the Crossplane CLI creates a `profile` in 
+`.crossplane/config.json` to cache unprivileged account information. 
+
+{{<hint "note" >}}
+The `session` field of `config.json` file is a session cookie identifier. 
+
+The `session` value isn't used for authentication. This isn't a `token`.
+{{< /hint >}}
+
+#### Authenticate with a registered Upbound organization
+
+Authenticate to a registered organization in the Upbound Marketplace with the 
+`--account` option, along with the username and password or token. 
+
+For example, 
+`crossplane xpkg login --account=Upbound --username=my-user --password -`.
+
+### xpkg logout
+
+Use `crossplane xpkg logout` to invalidate the current `crossplane xpkg login` 
+session.
+
+{{< hint "note" >}}
+Using `crossplane xpkg logout` removes the `session` from the 
+`~/.crossplane/config.json` file, but doesn't delete the configuration file.
+{{< /hint >}}
+
+### xpkg push
+
+Push a Crossplane package file to a package registry. 
+
+The Crossplane CLI pushes images to the 
+[Upbound Marketplace](https://marketplace.upbound.io/) at `xpkg.upbound.io` by 
+default.
+
+{{< hint "note" >}}
+Pushing a package may require authentication with 
+[`crossplane xpkg login`](#xpkg-login)
+{{< /hint >}}
+
+Specify the organization, package name and tag with  
+`crossplane xpkg push <package>`
+
+By default the command looks in the current directory for a single `.xpkg` file 
+to push. 
+
+To push multiple files or to specify a specific `.xpkg` file use the `-f` flag.
+
+For example, to push a local package named `my-package` to 
+`crossplane-docs/my-package:v0.14.0` use:
+
+`crossplane xpkg push -f my-package.xpkg crossplane-docs/my-package:v0.14.0`
+
+#### Flags
+
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag              | Description                                   |
+| ------------ | -------------          | ------------------------------                |
+| `-f`         | `--package-files=PATH` | A comma-separated list of xpkg files to push. |
+{{< /table >}}
+
+For example, to push a package named `my-configuration` use  
```

Comment:
does this example add more value that hasn't already been shown? everything this does is covered by the previous example.

also, would it work OK? there isn't a registry, org, repo, or tag specified 🤔